build-gluster-org/jobs/32-bit-build-smoke.yml

```diff
@@ -1,6 +1,6 @@
 - job:
     name: gh_32-bit-build-smoke
-    node: fedora30
+    node: fedora32
```

Comment:
So now I am reading that, I wonder if Fedora 30 is not the last Fedora with a 32 bits build. Have the jobs been tested on fedora 32 ?

Reply:
I have not tested it yet. I can give it a try.

Reply:
https://build.gluster.org/view/Github%20jobs/job/gh_32-bit-build-smoke/1176/, it failed. It has a i386 mock config for F32 but the job is failing at installing gluster.

Reply:
It was missing libuuid and rpcgen packages.